frontend/src/store/settings.ts

```diff
@@ -10,11 +10,13 @@ const settings = {
   advancedMode: 'advanced-mode',
   lastWallet: 'last-wallet',
   language: 'language',
+  sendHistorySave: 'send-history-save',
 };
 
 // Shared state between instances
 const isDark = ref(false); // true if user has dark mode turned on
 const advancedMode = ref(false); // true if user has advanced mode turned on
+const sendHistorySave = ref(true); // true if user send history is saved false if it is not
```

Comment:
(no RR so you can self-resolve)
```suggestion
const sendHistorySave = ref(true); // true if user send history is saved, false if it is not
```

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/560/commits/a972500d586021cd3283b25cd06b65658f0569a5